astroid/nodes/node_classes.py

```diff
@@ -4963,9 +4963,9 @@ class Unknown(_base_nodes.AssignTypeNode):
 
     def __init__(
         self,
+        parent: NodeNG,
         lineno: None = None,
         col_offset: None = None,
-        parent: None = None,
         *,
```

Comment:
Should we consider this a breaking change ?

Reply:
You are right, it is.

Reply:
What is the procedure for handling this?

Reply:
We'll wait for 4.0 to release it, and in astroid we just add a mildly strongly worded changelog entry. Or maybe we're not going to, I'm sure @DanielNoord will have an opinion about it :)